db/migrate/20160829202740_devise_create_users.rb

```diff
@@ -0,0 +1,30 @@
+class DeviseCreateUsers < ActiveRecord::Migration
+  def up
+    execute(<<-SQL
+      CREATE TABLE "users" (
+        "id" INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+        "name" VARCHAR DEFAULT '' NOT NULL,
+        "email" VARCHAR DEFAULT '' NOT NULL,
```

Comment:
Do we really need to allow emails that long? Maybe cut it down so we don't have such a large UNIQUE index.